src/util.js

```diff
@@ -99,6 +99,19 @@ function getRole(name) {
     return config.roles.get(name);
 }
 
+/**
+ * Gets role info and adds it. Will not error if the role or member doesn't exist.
+ * @param {string} name - The name of the role.
+ * @param {import('discord.js'.GuildMemberResolvable)} memberResolvable - The member to add.
+ * @returns {false|Promise<import('discord.js').GuildMember>} Returns false if the role or member doesn't exist. Otherwise returns the result of adding a role.
+ */
+function getAndAddRole(name, memberResolvable) {
```

Comment:
Role name and then member? You are using it the other way around in the code though...

Reply:
Yeah I'm a three head